dart-impl/base/src/internal/domain_locality.c

```diff
@@ -285,30 +354,25 @@ dart_ret_t dart__base__locality__domain__child(
    * TODO: Optimize, currently using exhaustive search.
    */
   for (int sd = 0; sd < domain->num_domains; ++sd) {
-    if (dart__base__locality__domain__child(
-          &domain->domains[sd], subdomain_tag, subdomain_out)
+    if (dart__base__locality__domain__child_rec(
+          domain->children[sd], subdomain_tag, subdomain_out)
         == DART_OK) {
-      DART_LOG_TRACE("dart__base__locality__domain__child - "
-                     "domain:%s, subdomain_tag:%s -> %s",
-                     (NULL != domain) ? domain->domain_tag : "?",
-                     subdomain_tag, (*subdomain_out)->domain_tag);
       return DART_OK;
     }
   }
-  DART_LOG_TRACE("dart__base__locality__domain__child - "
-                 "no subdomain %s in %s", subdomain_tag, domain->domain_tag);
   *subdomain_out = NULL;
   return DART_ERR_NOTFOUND;
 }
 
 /**
- * Find common parent of specified domains.
+ * Find common parent (lowest common ancestor) of specified domains by
+ * their common domain tag prefix.
  */
 dart_ret_t dart__base__locality__domain__parent(
-  const dart_domain_locality_t   * domain_in,
-  const char                    ** subdomain_tags,
-  int                              num_subdomain_tags,
-  dart_domain_locality_t        ** domain_out)
+  const dart_domain_locality_t     * domain_in,
+  const char                      ** subdomain_tags,
+  int                                num_subdomain_tags,
+  dart_domain_locality_t          ** domain_out)
 {
```

Comment:
Ah well, whitespace changes in this PR are negligible compared to the overall diff.
The trick is to append `&w=0` in the URL of the github diff.